src/inmanta/__init__.py

```diff
@@ -16,7 +16,7 @@
     Contact: code@inmanta.com
 """
 
-COMPILER_VERSION = "2023.2"
+COMPILER_VERSION = "2023.3"
```

Comment:
I think we need to diverge the compiler versions now that we've diverged the branches. This is the first time we've diverged this early in the year, so our internal docs doesn't cover it yet. By far the easiest would be to already move to the 2024 range for OSS, but that may not be ideal. I'm not sure how else to move forward though.

For iso6 this version is definitely good.